infra/cifuzz/actions/entrypoint.py

```diff
@@ -37,35 +37,39 @@ def main():
     FUZZ_TIME: The length of time in seconds that fuzzers are to be run.
     GITHUB_REPOSITORY: The name of the Github repo that called this script.
     GITHUB_SHA: The commit SHA that triggered this script.
+    GITHUB_REF: The pull request reference that triggered this script.
 
```

Comment:
Do we need to describe GITHUB_EVENT_NAME?

Reply:
Added.